### PR TITLE
[ZPI-27][FE] Widok zgłoszeń

### PR DESCRIPTION
**Pull request by gcfbn**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ services:
     ports:
       - '8000:8000'
     environment:
-      FRONTEND.ADDRESS: http://frontend:3000
+      FRONTEND.ADDRESS: http://localhost:3000
```
[the remark this answers is not in the document]
Comment (gcfbn) [reply docker-compose.yml:38]: Yup, it caused CORS problems. I can maybe change the rules in the authorizer in Spring.

**Comment by Veermove** [comment]
image

I think the missing links are not the only issue concerning attachments visibility

**Comment by Veermove** [comment]
Additional notes:
* build without `zpi/frontend/.env` fails even when the same vars are defined in `zpi/.env`
* would be nice to set title page to something more meaningful 
image

**Comment by Veermove** [comment]
Also please merge into your branch this https://github.com/Veermove/zpi/pull/23 if you don't mind. Unless you have some comments to be solved for that pr, it's ready to merge

**Comment by Veermove** [comment]
image
and perhaps it would be possible to gray-out/block/disable specific criteria if there is not rating for it? 
or moving present ones to the top? 
or both?

**Comment by gcfbn** [comment]
> Additional notes:
* build without `zpi/frontend/.env` fails even when the same vars are defined in `zpi/.env`
* would be nice to set title page to something more meaningful 
image
 
For the first bullet point, I have no idea how to fix it. While I imagine it should be somehow possible when running from Docker Compose, I can't find a way to do it when running fe. dev server only.

**Review by Remmo1 — APPROVED** [review]

**Review by Veermove — APPROVED** [review]

**Pull request merged**